app.rb

```diff
@@ -40,15 +40,30 @@
 post '/portal' do
   domain = params[:email].partition('@').last
 
-  organization = WorkOS::Portal.create_organization(
-    domains: [domain],
-    name: domain.partition('.').first,
+  organizations = WorkOS::Portal.list_organizations(
+   domains: [domain],
   )
 
-  portal_link = WorkOS::Portal.generate_link(
-    intent: 'sso',
-    organization: organization.id
-  )
+  if organizations.data.length > 0
```

Comment:
is this the most idiomatic way to check for an non-empty array?

Reply:
Yeah probably without lodash

Reply:
empty? should do the trick

```rb
irb(main):001:0> [].empty?
=> true
```

Reply:
beautiful